api/src/integrations/nris/datasource.js

```diff
@@ -195,7 +185,7 @@ class NrisDataSource {
     newRecord.addedBy = (this.auth_payload && this.auth_payload.displayName) || '';
     newRecord.updatedBy = (this.auth_payload && this.auth_payload.displayName) || '';
 
-    newRecord.sourceSystemRef = 'nris';
+    newRecord.sourceSystemRef = 'nris-epd';
```

Comment:
We should be updating the swagger dataSourceType enum, as well as the task service type (currently 'nris'), and the getDataSourceConfig switch handler.

Reply:
Yeah good call.  The swagger is a bit behind, and only mentions orders and inspections, so I can update that as well.